src/Features/CSharp/Portable/InvertIf/CSharpInvertIfCodeRefactoringProvider.cs

```diff
@@ -242,8 +242,7 @@ protected override bool IsSingleStatementStatementRange(StatementRange statement
             }
 
             return isSingleStatement(statementRange.FirstStatement);
-
-            bool isSingleStatement(StatementSyntax statement)
+            static bool isSingleStatement(StatementSyntax statement)
```

Comment:
>isSingleStatement [](start = 24, length = 17)

Empty line seemed beneficial. 
Also, I think IDE naming convention is PascalCase for local functions. It hurts me to say that ;-)

Reply:
done.